SonarQube.Bootstrapper/SonarQube.Bootstrapper.csproj

```diff
@@ -67,7 +67,7 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="Resources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
+      <Generator>PublicResXFileCodeGenerator</Generator>
```

Comment:
Resources made public so the tests can check the correct errors are being logged.